scripts/local_testnet/setup_time.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Could we replace the existing code in `start_local_testnet.sh` with a call to this script?
It may be a bit of over-optimisation from first sight, but could avoid some maintenance issues later down the track, e.g. this would prevent us from forgetting to update `cancunTime` on the `Deneb` branch.

Reply:
Do you mean adding the command in `setup_time.sh` to `start_local_testnet.sh`? Because `start_local_testnet.sh` is to automatically start the local testnet, while the `setup_time.sh` is to manually start it. 

I do agree that the `setup_time.sh` should be removed to keep it clean. How about merging `setup_time.sh` into `setup.sh`?

Reply:
I meant to replace these lines in `start_local_testnet.sh`
https://github.com/sigp/lighthouse/blob/c0b6b92f2704198a2301fdbda0ca08c29ce7e99b/scripts/local_testnet/start_local_testnet.sh#L106-L110
with 
```
./setup_time.sh
```

So that we keep the "set up time" logic in one place rather than two. `unstable` now contains the Cancun changes, so you'll notice that the following would need to be updated in both places before the refactor:

```
CANCUN_TIME=$((GENESIS_TIME + (DENEB_FORK_EPOCH * 32 * SECONDS_PER_SLOT)))
echo $CANCUN_TIME
sed -i 's/"cancunTime".*$/"cancunTime": '"$CANCUN_TIME"',/g' $genesis_file
cat $genesis_file
```

Reply:
Thank you for the comment
I have replaced the commands with  `./setup_time.sh genesis.json` in the file `start_local_testnet.sh` in [cbc3de1](https://github.com/sigp/lighthouse/pull/4749/commits/cbc3de1a3e71d8f3f9b677f15f6d8a8511ad5900)
This also means we have to move the `case` function in https://github.com/sigp/lighthouse/pull/4749#discussion_r1343981838 to `setup_time.sh`

For now:
-To start the local testnet automatically, we run `./start_local_testnet.sh genesis.json`
-To start the local testnet manually, we run a few commands, including `./setup_time.sh genesis.json`

Does it look strange that the `genesis.json` has now to be called by both `start_local_testnet.sh` and `setup_time.sh`?

Reply:
Also added the CANCUN_TIME code in `setup_time.sh` in [cb74726](https://github.com/sigp/lighthouse/pull/4749/commits/cb74726bf739403b6ce883155a523207eb42c4cf)